index.html

```diff
@@ -2501,6 +2501,71 @@ <h3 id="applying">
     </section>
     <section>
       <h2>
+        Multi-purpose members
+      </h2>
+      <p>
+        This section specifies members that can be used on multiple objects.
+        Each member specifies with object a member can be used with.
+      </p>
+      <section>
+        <h3>
+          <code>platform</code> member
+        </h3>
+        <p>
+          The <dfn data-lt="application-platform"><code>platform</code>
+          member</dfn> represents the <a>platform</a> to which a containing
+          object applies.
+        </p>
+        <p>
+          The following object types can make use of this member:
+        </p>
+        <ul>
+          <li>
+            <a>Application object</a>
+          </li>
+          <li>
+            <a>Image object</a>
+          </li>
+        </ul>
+        <p>
+          A <dfn>platform</dfn> represents a software distribution ecosystem or
+          possibly an operating system.
+        </p>
+        <p class="note">
+          This specification does not define the particular values for a the
+          <a><code>platform</code> member</a>. However, the working group
+          maintains a <a href=
+          "https://github.com/w3c/manifest/wiki/Platforms">list of known
+          platform values</a> in our wiki.
```

Comment:
I suspect there will only be like 3 or 4. If it gets out of hand, we should add something.